app/screens/home/channel_list/servers/index.tsx

```diff
@@ -144,6 +144,7 @@ export default function Servers() {
             subscriptions.forEach((unreads) => {
                 unreads.subscription?.unsubscribe();
             });
+            subscriptions.clear();
```

Comment:
Just curiosity, what is subscriptions.clear() doing here?

Reply:
subscriptions is defined outside of the component at the file level, without clearing it on unmount, it will hold the previous values causing the badges to not show accurately